bin/check-uptime.rb

```diff
@@ -35,11 +36,18 @@ class CheckUptime < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i),
          default: 180
 
+  option :reverse,
```

Comment:
How about naming the option a little bit more intuitively so that a user does not even have to read the description to understand. I am thinking either `--reverse-operators` or `--reverse-comparison` would be better than `--reverse`.

Reply:
Another good option that would be very obvious would be `--greater-than` verse implies knowing what's there, `--greater-than` explains itself without inspecting the code and one could assume that with no `--less-than` parameter that would be the default behavior.